BunqSdk.Tests/Model/Generated/Object/NotificationUrlTest.cs

```diff
@@ -73,8 +77,15 @@ string referencedObjectPropertyName
             
             Assert.NotNull(model);
             Assert.NotNull(referencedModel);
-            Assert.IsType(classNameExpected, referencedModel);
-            Assert.Equal(classNameExpected, referencedModel.GetType());
+            Assert.IsType(classTypeExpected, referencedModel);
+            Assert.Equal(classTypeExpected, referencedModel.GetType());
```

Comment:
Looks like these two assertions (L80 & L81) do the exact same thing

Reply:
I'd personally go with `Assert.IsType()`. Just make sure it's consistent with the assertion below (L88)